packs/spell-effects/spell-effect-anima-invocation.json

```
@@ -44,11 +44,17 @@
                 "property": "property-runes",
                 "value": "ghost-touch"
             },
+            {
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "weapon-traits",
+                "value": "holy"
+            },
             {
```

```suggestion
            },
            {
```
The effect doesn't state this happens, so it shouldn't be added by us. We don't want to editorialize.